Gems/Atom/RPI/Code/Include/Atom/RPI.Reflect/Pass/RasterPassData.h

```diff
@@ -39,6 +39,10 @@ namespace AZ
             RHI::Scissor m_overrideScissor = RHI::Scissor::CreateNull();
 
             RHI::DrawListSortType m_drawListSortType = RHI::DrawListSortType::KeyThenDepth;
+
+            // Forces viewport and scissor to match width/height of output image at specified index.
+            // Does nothing if index is negative.
+            s32 m_viewportAndScissorTargetOutputIndex = -1;
```

Comment:
Why use a signed int instead of a boolean?

Reply:
To specify which of the outputs' dimension to use for the viewport/scissor
-1: Don't do anything
0: Use output[0] for viewport height/width
1: Use output[1] for viewport height/width
etc.

Reply:
oh I somehow missed that code.